src/pages/developer-tools/docker/index.md

```diff
@@ -12,10 +12,57 @@ Similar to Github, DockerHub offers a wide variety of images (Ubuntu and others)
 ### See also
 - Docker Compose : to create and manage several containers at the same time.
 
-### Links
+
+#### More Information:
 You can find plenty of informations in the following sites : 
 - <a href='https://www.docker.com/' target='_blank' rel='nofollow'>Docker website</a>
 - <a href='https://docs.docker.com/' target='_blank' rel='nofollow'>Docker docs</a>
 - <a href='https://hub.docker.com/' target='_blank' rel='nofollow'>DockerHub</a>
+[logo]: https://www.docker.com/sites/default/files/Whale%20Logo332_5.png "Docker"
+
+
+## Docker
+
+Docker is an open source project to automate the dployment of any software applications inside containers.  A container image is a lightwieght, stand-alone, execuateble package of a piece of software that includes everything need to run it.
```

Comment:
Description for the Docker already exists can you compare both and make it better? 

and move the content to its correct positions ??